connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java

```diff
@@ -18,15 +18,48 @@
 package org.apache.kafka.connect.runtime.standalone;
 
 import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.utils.Utils;
+import org.apache.kafka.connect.runtime.Worker;
 import org.apache.kafka.connect.runtime.WorkerConfig;
+import org.apache.kafka.connect.storage.OffsetBackingStore;
+import org.omg.SendingContext.RunTime;
 
 import java.util.Map;
 
 public class StandaloneConfig extends WorkerConfig {
     private static final ConfigDef CONFIG;
 
+    /*
+     * NOTE: DO NOT CHANGE EITHER CONFIG STRINGS OR THEIR JAVA VARIABLE NAMES AS
+     * THESE ARE PART OF THE PUBLIC API AND CHANGE WILL BREAK USER CODE.
+     */
+
+    public static final Class<? extends OffsetBackingStore> OFFSET_STORAGE_CLASS_DEFAULT;
+
+    /**
+     * <code>offset.storage.file.filename</code>
+     */
+    public static final String OFFSET_STORAGE_FILE_FILENAME_CONFIG = "offset.storage.file.filename";
+    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "name of file used by FileOffsetBackingStore to save data";
+
     static {
-        CONFIG = baseConfigDef();
+
+        try {
+            OFFSET_STORAGE_CLASS_DEFAULT = Class.forName("org.apache.kafka.connect.storage.FileOffsetBackingStore", true, Utils.getContextOrKafkaClassLoader()).asSubclass(OffsetBackingStore.class);
+        } catch (Exception e) {
+            throw new RuntimeException("org.apache.kafka.connect.storage.FileOffsetBackingStore cannot be found", e);
+        }
+
+        CONFIG = baseConfigDef()
+                .define(WorkerConfig.OFFSET_STORAGE_CLASS_CONFIG,
+                        ConfigDef.Type.CLASS,
+                        OFFSET_STORAGE_CLASS_DEFAULT,
+                        ConfigDef.Importance.HIGH,
+                        WorkerConfig.OFFSET_STORAGE_CLASS_CONFIG_DOC)
+                .define(OFFSET_STORAGE_FILE_FILENAME_CONFIG,
```

Comment:
This exists in the config now, but it doesn't look like `FileOffsetBackingStore` has been updated to use this configuration rather than extracting it itself? The goal for these changes was to centralize these options, ensure they are all listed when we generate docs, and ensure the options get validated all at once when we do initial configuration.

Seems like the same is true for the `KafkaOffsetBackingStore` options as well.